Icon for Voronoi diagram effect

Lehonti opened the pull request:
Made by myself using free software.

It doesn't need to be the final one, but I thought it would be good to have some depiction of the effect at least.

As for the licensing, I don't know what license to use. I'm certainly happy to give the icon away, but I think it would have to be explained in the README file? I guess that the project's MIT license applies to its own home-grown icons automatically, just like it does in the case of code files, but I would have to confirm that.

cameronwhite commented:
Looks fine to me 👍 
For the licensing question, I think the MIT license would apply by default unless otherwise specified, e.g. as in `Pinta.Resources/icons/hicolor/scalable/readme.md` where many icons come from third-party icon sets. I think the About dialog also lists such cases

Lehonti commented:
I see. I added some clarification (that some icons come from 'Pinta contributors') in the readme file and the 'about' dialog. Also, in the last commit I added file with the list of such icons. If that works licensing-wise, I'd say this PR is ready

cameronwhite commented:
That looks fine to me 👍
